### PR TITLE
Benchpress setup

### PR DESCRIPTION
**Pull request by tbosch**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -4,74 +4,97 @@ set -e -o pipefail
 
 cd `dirname $0`
 
-export NODE_PATH=${NODE_PATH}:$(pwd)/dist/all:$(pwd)/dist/tools
-
-
-rm -rf ./dist/all/
-mkdir -p ./dist/all/
+PACKAGES=(core
+  compiler
```
Comment (IgorMinar) [inline build.sh]: can you alphabetize this list please

File: modules/@angular/benchpress/package.json [hunk 2]
```diff
@@ -1,15 +1,13 @@
 {
   "name": "@angular/benchpress",
-  "version": "0.0.0-PLACEHOLDER",
+  "version": "0.1.0",
   "description": "Benchpress - a framework for e2e performance tests",
   "main": "index.js",
   "typings": "index.d.ts",
   "dependencies": {
-    "@angular/core": "0.0.0-PLACEHOLDER",
+    "@angular/core": "2.0.0-rc.7",
```
Comment (IgorMinar) [inline modules/@angular/benchpress/package.json]: do you really want to pin this to rc.7?

File: build.sh (hunk 1 above)
Comment (tbosch) [inline build.sh]: No, the order is important for dependencies

**Comment by evmar** [comment]
rebased as a2bf334e6ee9f6a8ee6982d091bcb4e40a9b8418

**Pull request closed without merging**